tests/expected_service_account.yaml

```diff
@@ -3,38 +3,38 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: relase-long-name-rel-chart-long-name-cha-value-long-service-nam-env
+  name: release-long-name-re-chart-long-name-cha-value-long-service-nam-env
```

Comment:
You shouldn't have these changes, already made on master, can you try to do a rebase?